ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -293,7 +293,8 @@ export default defineComponent({
             (
               !hasMinimumGroups() ||
               hasEmptyGroup.value ||
-              !hasUnsavedChanges.value ||
+              hasUnsavedChanges.value ||
```

Comment:
Is this line still required with the refactored condition?

Reply:
Nope, silly me